csrankings-g.csv

```diff
@@ -583,6 +583,7 @@ Gorjan Alagic,University of Maryland - College Park,http://www.alagic.org,NOSCHO
 Gorkem Serbes,Yıldız Technical University,https://www.ce.yildiz.edu.tr/personal/gorkem?lang=en,96vDpfcAAAAJ
 Görschwin Fey,Hamburg University of Technology,https://www.tuhh.de/es/ce/people/gfey.html,NOSCHOLARPAGE
 Gorur Narayana Srinivasa Prasanna,IIIT Bangalore,http://www.iiitb.ac.in/faculty_page.php?name=GNSrinivasaPrasanna,NOSCHOLARPAGE
+Gosse Bouma,University of Groningen,http://www.let.rug.nl/gosse/,sd-w59cAAAAJ

```

Comment:
Not clearly in CS (see inclusion criteria).

Reply:
removed